download/download.go

```diff
@@ -121,6 +125,20 @@ func (d *Downloader) SetCache(etag string) {
 	d.etag = etag
 }
 
+// Trigger can be used to control when the downloader attempts to download
+// a new bundle in manual triggering mode.
+func (d *Downloader) Trigger(ctx context.Context) {
+	done := make(chan struct{})
+	d.trigger <- done
+
+	select {
+	case <-done:
+		return
+	case <-ctx.Done():
```

Comment:
Is there a way to get this context passed to the HTTP client making the download request?  It'd be nice to have that get cancelled along with the context.

Reply:
Done.